bin/compile

```diff
@@ -219,6 +230,8 @@ COMPILE_CMD="${COMPILE_CMD:-}"
 NGINX_HTTP_INCLUDES="${NGINX_HTTP_INCLUDES:-}"
 NGINX_INCLUDES="${NGINX_INCLUDES:-}"
 ACCESS_LOG_FORMAT_COMPOSER_JSON="${ACCESS_LOG_FORMAT_COMPOSER_JSON:-}"
+DATADOG_TRACER_VERSION="${DATADOG_TRACER_VERSION:-latest}"
+DATADOG_APPSEC_VERSION="${DATADOG_APPSEC_VERSION:-0}"
```

Comment:
Not sure if those environment variables should start with `DATADOG_` or `DD_`.

I think `DD_` is less explicit, but it would be consistent with other Datadog environment variables (like `DD_APPSEC_ENABLED`).